ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/dot1x.j2

```diff
@@ -13,11 +13,13 @@
 
 ### 802.1X Interfaces
 
-| Interface | State | Phone Force Authorized |
-| --------- | ----- | ---------------------- |
+| Interface | State | Phone Force Authorized | Pae Mode | Reauthentication |
```

Comment:
Minor comments:
- PAE should be in caps since it is an abbreviation.
- The "PAE Mode" seems like one of the most important knobs to even turn on 802.1x, so IMO we should move it to the second column.

Reply:
Updated per your request